### PR TITLE
Add new feature to count messages by date

### PR DESCRIPTION
**Pull request by WLSF**
### Description

Added 2 new functions on `reporting.ex`:
1. count_sent_messages/0 (fetch messages that contains user_id, grouped by date)
2. count_received_messages/0 (fetch messages that contains customer_id, grouped by date)

### Issue

#255 

### Screenshots

N/A updates on frontend

## Checklist

- [x] Everything passes when running `mix test`
- [x] Ran `mix format`
- [x] No frontend compilation warnings

### CODE REVIEW

**Review by reichert621 — CHANGES_REQUESTED** [review]
looks great @WLSF! just one small request to give the methods slightly more descriptive names 😄

File: lib/chat_api/reporting.ex [hunk 1]
```diff
@@ -31,6 +31,20 @@ defmodule ChatApi.Reporting do
     |> Repo.all()
   end
 
+  def count_sent_messages() do
```
[the remark this answers is not in the document]
Comment (WLSF) [reply lib/chat_api/reporting.ex:34]: hmm, makes sense! thanks!

**Review by reichert621 — APPROVED** [review]
awesome, thanks for the quick turnaround on this! 🚀

**Comment by reichert621** [comment]
let me know if there are any other tickets you'd be interested in tackling @WLSF 😁

**Pull request merged**